en/option/partial/emphasis-common.md

```diff
@@ -45,6 +45,9 @@ When the data is highlighted, whether to fade out of other data to focus the hig
 + `'ancestor'` Focus on all ancestor nodes.
 + `'descendant'` Focus on all descendants nodes.
 {{ /if }}
+{{ if: ${hasRelative} }}
++ `'relative'` since `v${version}`, focus on all ancestor and descendants nodes.
```

Comment:
Could we move the version hint to the end of the sentence and wrap it in parentheses? The same to Chinese documentation.